documentation/md/getting-started.md

```diff
@@ -4,10 +4,14 @@ This section will familiarise you with the basic steps necessary to start using
 
 ## Including Cytoscape.js
 
-If you are using a simple HTML environment (without a build system), then include Cytoscape.js in a `<script>` tag, e.g.:
+If you are using a simple HTML environment (without a build system), then source Cytoscape.js in a `<script>` tag or [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) it as an ES6 module, e.g.:
 
 ```html
 <script src="cytoscape.min.js"></script>
+<!--Or as an ES6 module-->
```

Comment:
The second example should probably be in a separate code block to indicate it's a different example.

The `import` line shouldn't include the `dist` part, or people will think they can link directly to the repo.